openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -753,7 +758,8 @@ def test_register_form_third_party_auth_running_google(self, input_country_code,
         with simulate_running_pipeline(
             "openedx.core.djangoapps.user_authn.views.login_form.third_party_auth.pipeline", "google-oauth2",
             email="bob@example.com",
-            fullname="Bob",
+            first_name="Bob",
+            last_name='Joe',
```

Comment:
does this pipeline actually return `first_name` and `last_name`?

Reply:
as its not part of your PR to have pipeline changes, I would say let's not change pipeline tests and let them fail